src/store/store.js

```diff
@@ -0,0 +1,65 @@
+import {makeAutoObservable} from 'mobx';
+class Store {
+  categories = [];
+  questionSets;
+  questions;
+  score = 0;
+  BASE_URL = 'http://192.168.100.106:1339';
+
+  constructor() {
+    (async () => {
+      await this.fetchCategories();
+      await this.fetchQuestionSets();
+      await this.fetchQuestions();
+    })();
```

Comment:
I'm not sure, do you really need to do this inside of constructor? Maybe it's better to call it inside of App, I'm not sure.